code/_onclick/hud/robot.dm

```diff
@@ -276,6 +276,8 @@ var/obj/screen/robot_inventory
 		r.client.screen -= r.robot_modules_background
 
 /mob/living/silicon/robot/update_hud()
-	..()
+	if(ui_style_vr) //VOREStation Edit
+		hands.icon = 'icons/mob/screen1_robot_vr.dmi'
 	if(modtype)
 		hands.icon_state = lowertext(modtype)
+	..()
```

Comment:
Should probably put //VOREStation Edit comments on this line too, or better yet, comment out the return at the top and leave it with an edit comment.

Or better-better-yet, put this in a crazy sideways override in the robot_vr.dm file and make sure to call ..() after